.github/workflows/check-links.yml

```diff
@@ -0,0 +1,227 @@
+# This workflow checks for broken links in the built site using htmltest
+#
+# Features:
+# - Runs weekly every Monday at 9:00 UTC
+# - Can be manually triggered
+# - Uses htmltest (same as @cncf/techdocs) for link checking
+# - Creates/updates GitHub issues when broken links are found
+# - Automatically closes issues when all links are fixed
+# - Uploads detailed results as artifacts for review
+#
+# Configuration:
+# - Uses .htmltest.yml for skip patterns and other settings
+# - Uses Makefile (aligned with @cncf/techdocs) for build orchestration
+# - Skips social media links to avoid rate limiting
+# - Maintains a reference cache to speed up checks
+
+name: Check Links on Live Site
+
+on:
+  schedule:
+    # Run every Monday at 9:00 UTC
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build site
+        run: npm run build
+
+      - name: Check links with htmltest
+        id: htmltest
+        continue-on-error: true
+        run: |
+          echo "Running link check with htmltest..."
+          make check-links 2>&1 | tee htmltest-output.txt
+          exit ${PIPESTATUS[0]}
+
+      - name: Process and analyze results
+        id: analyze
+        if: always()
+        run: |
+          cat > analyze-htmltest.js << 'EOFJS'
+          const fs = require('fs');
+          const path = require('path');
```

Comment:
Why not put the .js content in a JS file rather than cat'ing it? You can put it under `.github/scripts`